### ISSUE TITLE
Is this project dead. Add collaborators ?

### ISSUE THREAD

**Issue by kaareal** [AvianFlu/ntwitter#100]
It has been 5 month since the latest update and the library is not compatible with twitters 1.1 rest api. Plenty of good pull requests. 

https://github.com/AvianFlu/ntwitter/pull/95
https://github.com/AvianFlu/ntwitter/pull/88
https://github.com/AvianFlu/ntwitter/pull/79

But nobody is pulling.

So following the history of node-twitter. We are now on the 3 fork of the library.
https://github.com/technoweenie/twitter-node
https://github.com/jdub/node-twitter
https://github.com/AvianFlu/ntwitter
Do we need a 4. fork ? 

I think the best option would be to get @AvianFlu to add some collaborator.

**Comment by martindale** [comment]
I agree with this assessment, this is a pivotal library for a lot of my work and have been growing increasingly frustrated with the lack of management... @AvianFlu, I'd be happy to take over as a maintainer, let me know.

**Comment by marshallbu** [comment]
Another +1, I just recently forked to implement some of the pull requests

**Comment by passcod** [comment]
I had a need for a working Node client for Twitter 1.1 immediately so I’ve forked it and [published it to npm](https://npmjs.org/package/mtwitter) under the name **[mtwitter](/passcod/mtwitter)**. It “works for me™” although I haven’t tested it extensively. I pulled in a bunch of the PRs (just the straightforward ones, though; I'm not comfortable enough with the codebase just yet to merge more complex changes manually — but that's where _you_ come in!): #77, #95, #79, #94, #86, #82, and updated the list of contributors on the `package.json`.

> Do we need a 4th fork?

If @AvianFlu rises from the dead and resurrects this, it'd be cool, but so far (sadly) it looks like there is a need for a new fork, yes. However, I don't think I'll be spending much time maintaining this, and I'm keen to open up the process to be more community-oriented, so I'm quite happy to give push rights to someone(s), @martindale, if you're interested? (As well as publish rights / maintainer role on npm, of course :)

**Comment by kaareal** [comment]
Yeah I also dont have more time to wait, need to get started with the twitter 1.1 rest api migration. 
Gonna use @passcod mtwitter for now then. 

I would also be interested in getting added as a maintaining.
Thanks for getting this rolling

**Comment by marshallbu** [comment]
Can we also look into maybe pulling in #71?  I added it to my fork since I mostly use the streaming api, and cleans up some parts of the code here and there.

**Comment by passcod** [comment]
@marshallbu If you can clean it up and PR it in mtwitter I'll be happy to pull it. It looks significant so I don't want to touch it myself carelessly as I'm not too comfortable with the streaming API.

**Comment by marshallbu** [comment]
@passcod I'll be happy to do that, after a few shots of vodka.  Its not as many changes as it looks, there were a TON of changes where the submitter changed the tabbing on (way too many) lines, so it made it look immense.  I'll take a look at yours and see if I can make the same changes.

**Comment by AvianFlu** [comment]
Sorry, everybody!

Who wants commit access? I haven't had the time for this library for a while now.  I'll start trying to figure out some good people to add over the next few days.

**Comment by martindale** [comment]
Hey @AvianFlu!  Glad to have you back.  I'd love to have maintainer access, been merging changes from various forks for some time now for production deployments!

**Comment by passcod** [comment]
Oh, hey didn't see that. Would be good to give npm maintainer access to people too, @AvianFlu. I guess I'll close my fork now :)

**Comment by ricardobeat** [comment]
What's the status on this? I'm using https://github.com/ttezel/twit for now.

**Comment by marshallbu** [comment]
@ricardobeat some people have moved over to https://github.com/passcod/mtwitter for the time being

**Comment by martindale** [comment]
@AvianFlu what's going on?
